plugins/out_prometheus_remote_write/remote_write.c

```diff
@@ -332,6 +363,21 @@ static struct flb_config_map config_map[] = {
      0, FLB_TRUE, offsetof(struct prometheus_remote_write_context, http_passwd),
      "Set HTTP auth password"
     },
+#ifdef FLB_HAVE_SIGNV4
+#ifdef FLB_HAVE_AWS
+    {
+     FLB_CONFIG_MAP_BOOL, "aws_auth", "false",
+     0, FLB_TRUE, offsetof(struct prometheus_remote_write_context, has_aws_auth),
+     "Enable AWS SigV4 authentication"
+    },
+    {
+     FLB_CONFIG_MAP_STR, "aws_service", NULL,
```

Comment:
can we have a default value here? I bet for prom its always the same. User shouldn't need to configure this

Reply:
yep, added